README.md

```diff
@@ -51,3 +51,35 @@ Code coverage will be generated on path:
 ```
 {project-root}/{module}/target/scala-{scala_version}/jacoco/report/html
 ```
+## Health check endpoint
+Springboot Actuator enabled for this project. This provides the user with an endpoint (readable via http or JMX)
+that describes the status of the login-gateway as well as it's dependencies (for example: Databases, Ldap, server statistics, etc).
+### Accessing health endpoint via http
+Health Endpoint can be accessed via http using the following URL: `http://localhost:8080/actuator/health`
```

Comment:
```suggestion
Health Endpoint can be accessed via http using the following URL: `http://localhost:port/actuator/health`
```
8080 is not the correct port for us, in config we set 9090, but we could be generic and just write `port` (also once again below)